resources/intent/fokoos_base/tpu/fokoos_tpu_nozzle_0.40_layer_0.30_quick.inst.cfg

```diff
@@ -0,0 +1,55 @@
+[general]
+version = 4
+name = Quick
+definition = fokoos_base
+
+[metadata]
+setting_version = 20
+type = intent
+intent_category = FKS Fast
+quality_type = fokoos_layer_030
+material = generic_tpu
+variant = 0.40mm_fokoos_nozzle 
+
+[values]
+layer_height_0 = =0.3
+
+wall_thickness = =line_width*2
+wall_0_wipe_dist = 0.2
+z_seam_type = sharpest_corner 
+
+top_bottom_thickness = =layer_height*3
+
+infill_sparse_density = 16
+infill_pattern = ="grid"
+infill_overlap = 10
+infill_before_walls = true
+
+material_print_temperature = 230
+
+speed_print = 300
+speed_infill = =speed_print
+speed_travel = 420
+speed_layer_0 = 150
+speed_wall_x = =speed_print
+speed_travel_layer_0 =250
+speed_z_hop =10
+
+retraction_speed = 300
+retraction_retract_speed = =retraction_speed
+retraction_prime_speed = =retraction_speed
+retraction_count_max = 90
+retraction_combing = ="all"
+
+cool_fan_full_at_height = =layer_height_0
+cool_min_layer_time = 5
+cool_min_speed = 30
+
+skirt_line_count = 1
+skirt_gap = 3
+
+meshfix_maximum_resolution = 0.2
+multiple_mesh_overlap = 0.2
+
+minimum_polygon_circumference = 1.6
```

Comment:
Are you very sure that this should be set in a profile? Looking at the description, it doesn't make a lot of sense to me to do that...

Reply:
delete it